script.js

```diff
@@ -0,0 +1,54 @@
+const addBtn = document.getElementById('add');
+const booksList = [];
+
+addBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  const title = document.getElementById('title');
+  const author = document.getElementById('author');
+  const book = {
+    Title: title.value,
+    Author: author.value,
+  };
+  const list = document.getElementById('book-list');
+  const addDiv = document.createElement('div');
+  addDiv.classList.add('book');
+  booksList.push(book);
+  localStorage.setItem('books', JSON.stringify(booksList));
+  addDiv.innerHTML += `
+  <p>${book.Title}</p>
+  <p>${book.Author}</p>
+  <button class="delete button">Remove</button>
+  <hr>
+  `;
+  list.appendChild(addDiv);
+  document.getElementById('title').value = '';
+  document.getElementById('author').value = '';
+});
+
```

Comment:
- [ ] There is an issue with your application. When I add two books and refresh the page, only the last book appears on the page. This is because when a book is added you set the local storage item to be the newly added book, thereby removing the other ones.

  I think you can fix this but updating the bookList variable to load the data from the local storage instead of using an empty array.

  ```
  const booksList = JSON.parse(localStorage.getItem('books'));
  ```

  Two books are added | The page is refreshed
  :---:|:---:
  ![firs](https://user-images.githubusercontent.com/81468675/191012648-0efc377e-ed3f-455a-b43e-070a8982ace0.jpg) | ![sec](https://user-images.githubusercontent.com/81468675/191012642-6e6328bd-b233-491c-bdea-43ba52cb3a2f.jpg)

Reply:
thanks for the advice it worked